packages/web-app-files/tests/unit/mixins/actions/delete.spec.js

```diff
@@ -15,6 +15,14 @@ const Component = {
   mixins: [Delete]
 }
 
+const store = new Vuex.Store({
+  getters: {
+    capabilities: () => {
+      return {}
```

Comment:
My gripe with this is that it doesn't check the newly added logic via `this.capabilities?.files?.permanent_deletion` (as in it should now test both cases, one where permanent_deletion is true and one where it's false)